src/com/flowers/Chamomile.java

```diff
@@ -0,0 +1,28 @@
+package com.flowers;
```

Comment:
Let's make the root package for example. com.flower.kiev.
And put all flower-classes into the package entity or domain, and class FlowerStore into package service